mappings/net/minecraft/util/WoodType.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4719 net/minecraft/util/SignType
+CLASS net/minecraft/class_4719 net/minecraft/util/WoodType
```

Comment:
Is the used for woods, in general? Also maybe this should be in block package as well...

Reply:
Well it's certainly not just for signs, as it also has fence gate stuff, and a `BlockSetType`.